doc/pages/administrator/030-installation_kubernetes.md

```diff
@@ -1,41 +1,93 @@
 # Installing Astarte Operator
 
-Operator installation is usually intermediated by [`astartectl`](https://github.com/astarte-platform/astartectl),
-through the `astartectl cluster` subcommand. `astartectl cluster` manages the entire lifecycle of Astarte within
-a single Kubernetes Cluster, with regards of both the Operator and the Astarte instance(s) that will be deployed
-in the Cluster.
+The most simple and common installation procedure exploits the Astarte Operator's
+[`Helm chart`](https://artifacthub.io/packages/helm/astarte/astarte-operator).
+
+Helm is intended to be used as the operator's lifecycle management tool, thus make sure you are
+ready with a working [`Helm installation`](https://helm.sh/docs/intro/install/).
 
 ## Installation
 
 Installing the Operator is as simple as
 
 ```bash
-astartectl cluster install-operator
+helm repo add astarte https://helm.astarte-platform.org
+helm repo update
+helm install astarte-operator astarte/astarte-operator --version 1.0.0-alpha.1
 ```
 
-This command will take care of installing all needed components for the Operator to run. This includes all the
-RBAC roles, Custom Resource Definitions, and the Operator itself. The `--version` switch allows to specify a
-version to install - when not specified, the latest version will be installed instead.
+This command will take care of installing all needed components for the Operator to run. This
+includes all the RBAC roles, Custom Resource Definitions, and the Operator itself. The `--version`
```

Comment:
I know you specified this here but most people will go for copypaste, so that's another argument for removing --version from the command.